src/main/java/org/example/springbootintro/controller/BookController.java

```diff
@@ -67,7 +76,11 @@ public BookDto updateById(@PathVariable Long id,
 
     @GetMapping("/search")
     @ResponseStatus(HttpStatus.OK)
-    public List<BookDto> search(@Valid BookSearchParametersDto bookSearchParametersDto) {
-        return bookService.search(bookSearchParametersDto);
+    @Operation(summary = "Search books", description = "Search books by author or title or both")
+    public List<BookDto> search(
+            Pageable pageable,
+            @Valid BookSearchParametersDto bookSearchParametersDto
```

Comment:
having a search parameters documentation is usually very useful for the whole team